crates/aranya-runtime/src/client/init.rs

```diff
@@ -0,0 +1,64 @@
+//! Init commands used to create new graphs.
+//! See [`crate::ClientState::new_graph`]
+
+use buggy::{bug, Bug};
+use serde::{Deserialize, Serialize};
+
+use crate::{Address, Command, CommandId, GraphId, Prior, Priority};
+
+#[derive(Serialize, Deserialize)]
+/// Used for serializing init commands
+pub(crate) struct InitCommand<'a> {
+    storage_id: GraphId,
+    id: CommandId,
+    #[serde(borrow)]
+    data: &'a [u8],
+    #[serde(borrow)]
+    policy: &'a [u8],
+}
+
+pub(crate) type InitCmd = Vec<u8>;
```

Comment:
If this type is going to be used by other crates in the `aranya` repo, it will need to be `pub`:
```suggestion
pub type InitCmd = Vec<u8>;
```